tools/ClassGen/NodeGen.cpp

```diff
@@ -82,7 +82,7 @@ int main(int argc, char **argv) {
       .addInput("Input")
       .addMember(MemberType::SizeT, "Kernel")
       .addMember(MemberType::SizeT, "Stride")
-      .addMember(MemberType::SizeT, "Pad")
+      .addMember(MemberType::VectorSizeT, "Pads")
       .addResultFromCtorArg()
       .addGradient()
       .setDocstring("Performs a Max Pool operation on the Input given provided "
```

Comment:
nit: Update the comment here to say Pads? Same on PoolAvg below too.

Reply:
Comments updated!